<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QObjectEditor</class>
 <widget class="QDialog" name="QObjectEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>535</width>
    <height>505</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>535</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Справочник объектов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/G.png</normaloff>:/images/G.png</iconset>
  </property>
  <widget class="QTabWidget" name="tab">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>521</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="actorsTab">
    <attribute name="title">
     <string>Акторы</string>
    </attribute>
    <widget class="QListWidget" name="actorList">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="predicatesTab">
    <attribute name="title">
     <string>Предикаты</string>
    </attribute>
    <widget class="QListWidget" name="predicateList">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="iactorsTab">
    <attribute name="title">
     <string>I-Акторы</string>
    </attribute>
    <widget class="QListWidget" name="inlineActorList">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ipredicatesTab">
    <attribute name="title">
     <string>I-Предикаты</string>
    </attribute>
    <widget class="QListWidget" name="inlinePredicateList">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>470</y>
     <width>511</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tab</tabstop>
  <tabstop>actorList</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>predicateList</tabstop>
  <tabstop>inlineActorList</tabstop>
  <tabstop>inlinePredicateList</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QObjectEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QObjectEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
